### DISCUSSION TITLE
How can i render trading view web using CefSharp v49.0. Im succesfully using this with my .Net 4.0 Project. But if im open trading view / charview web the browser only show white screen

### DISCUSSION THREAD

**Discussion by ekoprassetyo** [cefsharp/CefSharp#4341] [General]

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related  questions.

The version you are using is likely too old. Upgrade to a newer version.

**Reply by ekoprassetyo** [reply]
Okey im sorry, but i can't upgrade the version because my project using .net 4.0. It is possible to add method or handler request to load trading view in my cefsharp 49?

thank you for your kind

**Reply by amaitland** [reply]
> because my project using .net 4.0

Upgrade to .Net 4.5.2 or greater to use a newer version of CefSharp.

>  It is possible to add method or handler request to load trading view in my cefsharp 49?

No idea, you'd have to debug that yourself. I've no idea what trading view is.

Version is too old for me to provide you with any sort of free support.

**Reply by ekoprassetyo** [reply]
i know but i can't by my existing project is using .net 4.0, and im using cefsharp for webview to render my javascript chartview/trading view. but its only rendering white screen. and when i look error in console devtools its said 'Uncaught Syntax Error / Unexpected Identifier'

**Comment by BuslikDrev** [comment]
Вам нужно программирование изучать. Скорее всего какой-то скрипт не загружается из-за неправильного пути или другой ошибки в javascript.
Если захотите обновить CefSharp у вас ещё появятся проблемы, например, с загрузкой изображений с компьютера - будет ругаться на отсутствие https и не загружать файлы (установить псевдо домен: https://github.com/cefsharp/CefSharp/wiki/General-Usage#scheme-handler).
